sdk/package.json

```diff
@@ -1,53 +1,76 @@
 {
-    "name": "imagekitio-angular",
-    "version": "5.1.0",
-    "license": "MIT",
-    "scripts": {
-        "ng": "ng",
-        "start": "ng serve",
-        "build:app": "ng build",
-        "build": "ng build && npm run assets",
-        "test": "ng test",
-        "test:ci": "ng test --watch=false --code--coverage && codecov",
-        "test:e2e": "cypress open",
-        "lint": "ng lint",
-        "watch": "ng build --watch --configuration development",
-        "postinstall": "./scripts/postinstall.sh",
-        "assets": "node ./scripts/copy-assets.js",
-        "serve:test-app": "export SKIP_PREFLIGHT_CHECK=true;ng serve & cd ../samples/sample-server && npm run server"
-    },
-    "private": false,
-    "dependencies": {
-        "@angular/animations": "~12.2.0",
-        "@angular/common": "~12.2.0",
-        "@angular/compiler": "~12.2.0",
-        "@angular/core": "~12.2.0",
-        "@angular/forms": "~12.2.0",
-        "@angular/platform-browser": "~12.2.0",
-        "@angular/platform-browser-dynamic": "~12.2.0",
-        "@angular/router": "~12.2.0",
-        "imagekit-javascript": "^3.0.2",
-        "rxjs": "~6.6.0",
-        "tslib": "^2.3.0",
-        "zone.js": "~0.11.4"
-    },
-    "devDependencies": {
-        "@angular-devkit/build-angular": "~12.2.18",
-        "@angular/cli": "~12.2.18",
-        "@angular/compiler-cli": "~12.2.0",
-        "@types/jasmine": "~3.8.0",
-        "@types/node": "^12.11.1",
-        "codecov": "^3.8.3",
-        "cypress": "^13.7.2",
-        "cypress-file-upload": "^5.0.8",
-        "cypress-log-to-output": "^1.1.2",
-        "jasmine-core": "~3.8.0",
-        "karma": "~6.3.0",
-        "karma-chrome-launcher": "~3.1.0",
-        "karma-coverage": "~2.0.3",
-        "karma-jasmine": "~4.0.0",
-        "karma-jasmine-html-reporter": "~1.7.0",
-        "ng-packagr": "^12.1.1",
-        "typescript": "~4.3.5"
-    }
+  "name": "@imagekit/angular",
+  "version": "6.0.0",
+  "description": "Angular SDK for ImageKit.io - Image and video optimization, transformation, and delivery platform",
+  "keywords": [
+    "imagekit",
+    "imagekit-angular",
+    "image-optimization",
+    "image-transformation",
+    "cdn",
+    "image-cdn",
+    "video-cdn",
+    "angular",
+    "responsive-images",
+    "lazy-loading"
+  ],
+  "author": "ImageKit",
+  "license": "MIT",
+  "homepage": "https://imagekit.io",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/imagekit-developer/imagekit-angular.git"
+  },
+  "bugs": {
+    "url": "https://github.com/imagekit-developer/imagekit-angular/issues"
+  },
+  "private": false,
+  "peerDependencies": {
+    "@angular/common": ">=12.2.0",
+    "@angular/core": ">=12.2.0",
+    "@imagekit/javascript": "^5.1.0"
```

Comment:
`@imagekit/javascript` needs to be in `dependencies`. That's it. It is in peer and dev dependencies at the moment. Makes no sense. Check react pacakge.json https://github.com/imagekit-developer/imagekit-react/blob/master/package.json